src/utils.jl

```diff
@@ -1,3 +1,5 @@
+using Logging
+
```

Comment:
why is this needed?

Reply:
I might have needed it for debugging at some point.

I see Logging being used elsewhere, but I don't see it needed here.  I'll take it out.